### DISCUSSION TITLE
Run in Java 17

### DISCUSSION THREAD

**Discussion by gfengster** [nosan/embedded-cassandra#366] [Q&A] [answered]
Version 4.1.0 cannot run in Java 17. The error is 
```
Caused by: java.io.IOException: 'UnixCassandraDatabase{process=Process[pid=58783, exitValue=1]}' is not alive. Please see logs for more details.
Errors:
OpenJDK 64-Bit Server VM warning: Option UseBiasedLocking was deprecated in version 15.0 and will likely be removed in a future release.
Unrecognized VM option 'UseConcMarkSweepGC'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
Output:
	at com.github.nosan.embedded.cassandra.DefaultCassandra.await(DefaultCassandra.java:235)
	... 5 more
```

**Comment by nosan** [comment, marked as answer]
Apache Cassandra itself does not support Java 17. 

> ./cassandra -f      
OpenJDK 64-Bit Server VM warning: Option UseBiasedLocking was deprecated in version 15.0 and will likely be removed in a future release.
Unrecognized VM option 'UseConcMarkSweepGC'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
